### ISSUE TITLE
Segfaults on Debian Jessie

### ISSUE THREAD

**Issue by chinese-soup** [u8sand/Baka-MPlayer#201]
Hello.
I am running Debian jessie.
I am getting random segfaults upon loading some files, sometimes the same files open correctly and sometimes they don't. Changing the volume while having a file that actually plays open always causes a segfault.

**ldd /usr/bin/baka-mplayer**
[https://gist.github.com/chinese-soup/fedb519eba824fa53552](https://gist.github.com/chinese-soup/fedb519eba824fa53552)

**strace -o trace2 baka-mplayer**
[https://gist.github.com/chinese-soup/d37a9b6bbcb566453a7a](https://gist.github.com/chinese-soup/d37a9b6bbcb566453a7a)

**Comment by u8sand** [comment]
@chinese-soup are you able to compile baka-mplayer on Debian or do you need us to try and get you a binary?

The new version of baka outputs logs to stdout allowing us to directly obtain mpv debugging information, that information would reveal the error much more quickly than `strace`.

**Comment by petri152** [comment]
Hey, I'm having the exact same issue. I've used the pre-compiled binary from the webpage and built from source, same issues.

**Comment by u8sand** [comment]
@petri152 just to confirm those files don't crash with pure mpv. There is certainly something strange going on here, I seem to recall when I was getting it compiled on debian, the ones they had built had issues while the one I was able to compile myself was fine.. Maybe that is still present.

I think it's certainly time I get my hands on debian and try this out.

**Comment by petri152** [comment]
@u8sand Yeah, they work fine with mpv. Also, I should note, it worked fine on an installation of Debian I had last week but I needed to reinstall and it started happening. On this older installation the pre-compiled binary would crash but when I compiled it myself it would work fine.

**Comment by SEAPUNK** [comment]
I can confirm this issue, BTW. I'm on Arch Linux. Baka-MPlayer 2.0.4.

`ldd /usr/bin/baka-mplayer`
https://gist.github.com/SEAPUNK/827d1068224f38fbac55bd2f62535c9d

**Comment by ConundrumAdInfinitum** [comment]
Same on Windows 10 Pro x64 FYI
